src/main/fc/tasks.c

```diff
@@ -322,6 +322,9 @@ void tasksInit(void)
         } else if (rxRuntimeState.serialrxProvider == SERIALRX_CRSF) {
             // Reschedule telemetry to 500hz, 2ms for CRSF
             rescheduleTask(TASK_TELEMETRY, TASK_PERIOD_HZ(500));
+        } else if (rxRuntimeState.serialrxProvider == SERIALRX_GHST) {
+            // Reschedule telemetry to 200Hz for GHST
+            rescheduleTask(TASK_TELEMETRY, TASK_PERIOD_HZ(200));
```

Comment:
Why is this being rescheduled - if sending of telemetry frames is done as part of the RX task, then there should be no need to run the telemetry task more often than the default. JETIEXBUS and CRSF are just using a problematic implementation and should be fixed.

Reply:
Good point, removed, was not required.